agcounts/pow2.py

```diff
@@ -35,15 +35,8 @@ def resample_to_30hz(
         if filter is None:
             logger.error("Interpolating needs MNE. Please install MNE.")
             raise ImportError("mne")
-        filtered = filter.filter_data(
-            data.transpose(),
-            sfreq=sample_rate,
-            l_freq=None,
-            h_freq=15,
-            verbose=30,
-        )
-        data = filter.resample(filtered, down=sample_rate / 30).transpose()
-        del filtered
+
+        data = filter.resample(data, down=sample_rate / 30, axis=0)
```

Comment:
Does `filter.resample(.)` apply the low pass filter with the 15 Hz cut-off?

Reply:
Yes resample takes care of filtering.